### PR TITLE
Merge release/keyvault-4.1.0 into master

### PR DESCRIPTION
**Pull request by heaths**
This needed a separate merge commit since there were lots of conflicts expected, and re-recordings were needed.

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs [hunk 1]
```diff
@@ -31,9 +31,9 @@ public enum ServiceVersion
             V7_0 = 0,
 
             /// <summary>
-            /// The Key Vault API version 7.1-preview.
+            /// They Key Vault API version 7.1.
```
Comment (christothes) [inline sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs:34]: They => The

Comment (christothes) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs:34]: looks like this is in a few other places as well.

**Review by christothes — APPROVED** [review]
Looks good, with the exception of the They/The typo.

File: sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs (hunk 1 above)
Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs:34]: Oops. Well, that shipped. They're all fixed moving forward. 🤦‍♂️

**Pull request merged**